uktrade/data-hub-frontend
# Replace export page with React

The release of this PR is dependent on yellow turning on a feature flag.

The unit test coverage also drops ever so slightly in the controller tests. With the new controller testing the one line using mocha was extremely difficult, instead it is covered in the new functional tests. 

Ignoring whitespace the PR is +536 −229. Sorry for the slightly large size. Part of this is down to the fact that there were no functional tests in the area and I've had to write new ones from scratch.

## Description of change

Render a new export page design with React. This rearranges the existing data and adds the 'Countries of no interest' field. 

## Test instructions

When you go to the export page of a company you will see the new React page as shown in the 'after' screenshot below. 

## Screenshots
### Before

![FireShot Capture 030 - Exports - ONEBUTTON LIMITED - Companies - DIT Data Hub_ - www datahub trade gov uk](https://user-images.githubusercontent.com/42253716/73369757-08a39300-42ab-11ea-9a14-de00b66739b1.png)

### After

![FireShot Capture 031 - Exports - One List Corp - Companies - DIT Data Hub - localhost](https://user-images.githubusercontent.com/42253716/73369824-253fcb00-42ab-11ea-8120-e3fdd275fa75.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
